Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -413,17 +415,24 @@ extension SubscriptDeclSyntax {
 }
 
 extension FunctionDeclSyntax {
-    
-    func model(with acl: String, declType: DeclType, funcsWithArgsHistory: [String]?, processed: Bool) -> Model {
+
+    func model(with acl: String, declType: DeclType, funcsWithArgsHistory: [String]?, modifiers: [String : Modifier]?, processed: Bool) -> Model {
         var isStatic = false
         if let modifiers = self.modifiers {
             isStatic = modifiers.isStatic
         }
-        
+
         let params = self.signature.input.parameterList.compactMap { $0.model(inInit: false, declType: declType) }
         let genericTypeParams = self.genericParameterClause?.genericParameterList.compactMap { $0.model(inInit: false) } ?? []
         let genericWhereClause = self.genericWhereClause?.description
-        
+
+        let modifier: Modifier
+        if let passedModifiers = modifiers {
```

Comment:
This should be moved to SwiftSyntaxExtension - 'Trivia.metadata(with annotation: String, in val: String)'